### PR TITLE
Sub-menu in the Operators page

### PR DESCRIPTION
**Pull request by renetapopova**
Replaces #562

### CODE REVIEW

**Review by AlexicaWright — COMMENTED** [review]
Thank you @renetapopova , lots of work! As per the discussion on the Trello card, I am on the fence whether this is an improvement or not. There are so many operators that one doesn't get a good overview even with a sub-menu. However, one do get a sense of how many there are and I guess that counts. Bottom line, I say let's merge this!

**Comment by renetapopova** [comment]
@davidoliverSP2 please review it

File: cypher/cypher-docs/src/docs/dev/execution-plan-groups/operators.asciidoc [hunk 1]
```diff
@@ -1,12 +1,104 @@
 [[execution-plans-operators]]
-= Operators
+= Execution plan operators in details
```
Comment (davidoliverSP2) [inline cypher/cypher-docs/src/docs/dev/execution-plan-groups/operators.asciidoc]: I think this should be 

> Execution plan operators in detail

**Review by davidoliverSP2 — CHANGES_REQUESTED** [review]
Hi @renetapopova - this looks good, but there is something odd going on with the index file in this section - if you build this and look at Chapter 7. Execution plans, there is a lot of content building, and the toc doesn't include this page.

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @renetapopova - this is excellent 👍 

There is still an oddity with the back and forward links, so I created https://github.com/neo-technology/neo4j-manual-modeling/pull/1122 to update the content map.

**Pull request merged**